DigitalLearningSolutions.Web/Controllers/TrackingSystem/CentreConfiguration/RegistrationPromptsController.cs

```diff
@@ -53,15 +52,15 @@ public IActionResult EditRegistrationPrompt(int promptNumber)
         [Route("{promptNumber}/Edit")]
         public IActionResult EditRegistrationPrompt(EditRegistrationPromptViewModel model, string action)
         {
-            if (action.StartsWith("delete") && TryGetAnswerIndexFromAction(action, out var index))
+            if (action.StartsWith("delete") && TryGetAnswerIndexFromDeleteAction(action, out var index))

```

Comment:
Can we make these action strings an enum or set of constants?

Reply:
You probably also want to refer to the constant length for the string operation on line 257 in the controller